concrete/src/Cache/Page/PageCache.php

```diff
@@ -66,13 +85,27 @@ public function shouldCheckCache(Request $req)
         return true;
     }
 
+    /**
+     * Send the cache-related HTTP headers for a page to the current response.
+     *
+     * @deprecated Response headers should be set by the deliver method.
```

Comment:
I think it's worth being as verbose as possible when deprecating methods. I know that 99% of the time I run into a `@deprecated` method I have a very limited idea of what I need to do to avoid using that deprecated path.

What if this were suuuuuuper verbose, something like:
```php
/**
 * @deprecated Headers are no longer set directly. Instead, use the `$pageCache->deliver()` method to retrieve a 
 *             response object and either return it from a controller method or, 
 *             if you must, use `$response->prepare($request)->send()`
 */

Reply:
Done.